pkg/datapath/iptables/iptables.go

```diff
@@ -286,6 +290,7 @@ func (m *IptablesManager) RemoveRules() {
 
 func ingressProxyRule(cmd, l4Match, markMatch, mark, port, name string) []string {
 	ret := []string{
+		"-w",
```

Comment:
`-w` without a duration will make it block forever if someone holds the lock indefinitely. In this case, it makes sense to log an error and crash as otherwise it will be difficult to debug why cilium-agent doesn't make a progress during its initialization.

Reply:
@brb what do you thing is the right timeout here? 60 seconds?

Reply:
Kubernetes uses 5sec (https://github.com/kubernetes/kubernetes/blob/master/pkg/util/iptables/iptables.go#L130). We could use this value as well.

Keep in mind that no all iptables versions support `-w`, so we need to do the feature detection.